### DISCUSSION TITLE
useOnline-Hook

### DISCUSSION THREAD

**Discussion by michaelhuber1937** [mantinedev/mantine#1577] [Feature requests]
I just needed a detection if the browser is online/offline, so I created this little hook. Maybe somebody thinks it is useful/good idea and mabye it finds its way into the framework:

```
import { useState } from 'react';

const useOnline = () => {
    const [online, setOnline] = useState(window.navigator.onLine);

    window.addEventListener('offline', () => {
        setOnline(false);
    });

    window.addEventListener('online', () => {
        setOnline(true);
    });

    return online;
};
```

**Comment by rtivital** [comment]
You can already get this piece of data from the `use-network` hook

**Reply by michaelhuber1937** [reply]
Thanks you so much! I didn't know that this hook already exists!
